integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ItMiiSampleHelper.java

```diff
@@ -75,9 +76,6 @@
   private String getModelImageName(String suffix) {
     return new StringBuffer(DOMAIN_IMAGES_REPO)
         .append("mii-")
-        .append(currentDateTime)
-        .append("-")
-        .append(domainNamespace)
```

Comment:
I assume it's this fix that is preventing the explosion of image names, but I thought this was needed to allow parallel runs?  Unless you have removed deletion/cleanup of these objects?

Reply:
(Also, if this is the main cause of the explosion of image names, is it truly necessary to alter the tag logic?)

Reply:
The image name was mii-<timestamp>-<namespace>-wlsv1:WLS-v1. It is changed to mii-<testClassName>-wlsv1:<timestamp>-WLS-v1. To prevent the explosion of the image name and allow parallel runs, to be consistent with other tests, the image name should be fixed and the image tag contains the timestamp.